sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp

```diff
@@ -0,0 +1,108 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out
+// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
For my education, can you please elaborate, why these env vars are needed?
What is VISALTO?

Reply:
so `VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1` are actually needed to get IGC to run with invoke_simd, all the tests have them. i'm not sure why it doens't work automatically but we do need to pass them

for `VISALTO`, i just copied it from another test but it looks like that's testing with some optimization flags for link time optimization IGC, so that seems to be more testing an IGC feature and not super relevant to us, so i'll remove this, thanks